books/RayTracingInOneWeekend.html

```diff
@@ -2759,8 +2759,8 @@
 <div class="together">
 We don’t need to simulate any of the inside of the camera. For the purposes of rendering an image
 outside the camera, that would be unnecessary complexity. Instead, I usually start rays from the
-surface of the lens, and send them toward a virtual film plane, by finding the projection of the
-film on the plane that is in focus (at the distance `focus_dist`).
+lens aperture, and send them toward the projection plane (`focus_dist` away), where everything on
```

Comment:
the `focus_dist` is the distance from the focus plane to the projection point. NOT the focus plane to projection plane

Reply:
Actually, I don't understand this sentence...

Reply:
I can swap out “projection plane” for “focus plane”. In this model, they are both one and the same, as are the focal length and focus distance. Also note that there's no “projection point” any more, but a projection _disk_, though I guess you could say the disk of projection points.

Reply:
Though I suppose you could say that the image/projection plane still exists, but where would it be? Behind the aperture? I think it's basically done away with in this model. As another argument that the image/projection plane and the focus plane have been unified, consider that the image sampling happens directly on the (what would you call it?) focus/image rectangle.

A figure would definitely help here.

Reply:
Actually, I was looking at the text and not the figures. What's really happened here is a bit of slight-of-hand. The image-in-front-of-projection-point model has been swapped out for the image-behind-the-projection-point, and then the back half is immediately dropped.

Reply:
Heh, the original is still messed up — it describes sending rays “toward the virtual film plane”, but in the figures, the “film” is behind the lens. Will rework again.